lib/livebook/config.ex

```diff
@@ -275,10 +275,7 @@ defmodule Livebook.Config do
   def logout_enabled?() do
     {_type, module, _key} = Livebook.Config.identity_provider()
 
-    identity_logout? =
-      Code.ensure_loaded?(module) and function_exported?(module, :logout, 2)
-
-    authentication().mode != :disabled or identity_logout?
```

Comment:
FTR `authentication().mode != :disabled` was intentional, the reasoning is that currently we have no way to sign out from Livebook after you authenticate with `LIVEBOOK_PASSWORD`. I'm fine either way though.

Reply:
I think they have to be different things because when using an identity provider, you still need to log in as admin. We may also want to add a "Logout" button to the menu of deployed apps, which will also trigger this. So having the button doing different things at different stages could be confusing, so I think tying it exclusively to identity for now is the right call.